config/app.default.php

```diff
@@ -529,6 +529,15 @@
         'default' => null,
     ],
 
+    /**
+     * Uncomment to define custom actions to load
+     * This way some action beahavior can be overridden
+     */
+    // 'Actions' => [
```

Comment:
I'd like more having the configured action in `Signup` namespace. See below for consideration on `createAction()` method.